tests/core/propertyTypes.test.js

```diff
@@ -156,6 +156,26 @@ suite('propertyTypes', function () {
     });
   });
 
+  suite('matrix4', function () {
+    var parse = propertyTypes.matrix4.parse;
+    var stringify = propertyTypes.matrix4.stringify;
+
+    test('parses matrix4', function () {
```

Comment:
I would test being able to pass a `THREE.Matrix4` or even a `THREE.Quaternion` and hopefully it would still work. I think perhaps it should always resolve a three.js object rather than an array.